### ISSUE TITLE
Running on Linux

### ISSUE THREAD

**Issue by arnepeine** [tanjera/infirmary-integrated#141]
Hello,

are there any instructions on how to run II on Linux (in particular Ubuntu 20.04)?

Was not successful in running it.
Thanks,
Arne

**Comment by tanjera** [comment]
Great question- let me put together instructions now. I just got on my Debian box (it's the closest distro to Ubuntu- will work the same) so I can get the instructions right.

**Comment by tanjera** [comment]
I just updated the main website's Download page to "[Download & Install](https://www.infirmary-integrated.com/download/)" and added installation instructions for Windows & Linux. Will still need to add instructions for MacOS.

This technically should solve this issue, so I will close this issue on GitHub. Know that I am using Linux a lot more these days and am making progress towards building installation packages for Infirmary. I aim to start with a Debian/Ubuntu .deb package, though it will probably be weeks before I complete the task.

Your problem was probably the dependency on the .NET runtime- Infirmary only runs cross-platform because it runs on the cross-platform implementation of Microsoft's .NET platform. On an otherwise base installation of Debian (which is mostly identical to Ubuntu as far as Linux distributions go) with the .NET runtime installed, I was successful in launching Infirmary Integrated.

If you have any problems opening/running on Linux, feel free to either comment or send me a message. I'd be more than happy to help you get it up and running.

**Comment by tanjera** [comment]
I noticed an error in my last post. The .NET runtime does **not** need to be installed for Infirmary Integrated to run. I started working on building a Debian/Ubuntu package and realized I didn't have the runtime installed (I thought I did!) and Infirmary was still running just fine.

I didn't realize dotnet was including the runtime library files in my build, which is good by me. I will leave the instructions as is (instructing people to install the runtime manually) in case I ever build packages that don't include the runtime.
